app/server/api/vendor_payout_api.py

```diff
@@ -136,62 +154,69 @@ def post(self):
             csv_data = post_data.get('csv_data', [])
             f = io.StringIO(csv_data)
             reader = csv.DictReader(f)
+
         transfers = []
         for line in reader:
-            vendor = db.session.query(TransferAccount).filter(TransferAccount.id == line['ID']).first()
-            if not vendor.primary_user.has_vendor_role:
-                response_object = {
-                    'message': str(f'{vendor} is not a vendor!'),
-                }
-                return make_response(jsonify(response_object)), 400
-
+            tid = line['Transfer ID']
+            transfer = db.session.query(CreditTransfer).filter(CreditTransfer.id == tid).first()
+            message = ''
 
-            transfer = db.session.query(CreditTransfer).filter(CreditTransfer.id == line['Transaction ID']).first()
             if not transfer:
-                response_object = {
-                    'message': f'Tranfer with ID {line["Transaction ID"]} not found!'
-                }
-                return make_response(jsonify(response_object)), 400
-
-            if round(transfer.transfer_amount) != round(dollars_to_cents(line["UnitAmount"])):
-                response_object = {
-                    'message': f'Invalid transfer amount!'
-                }
-                return make_response(jsonify(response_object)), 400
```

Comment:
the problem with returning a 400 (or 500ing as per earlier), is that it creates a missleading response - if this transfer is the 4th one in the list, then the previous 3 will have already succeeded